frontend/www/js/omegaup/problem/edit.js

```diff
@@ -571,34 +611,35 @@ OmegaUp.on('ready', function() {
         T.problemEditGoToProblem +
         '</a>',
     );
-    $('input[name=title]').val(problem.title);
+
     $('#statement-preview .title').html(UI.escape(problem.title));
-    $('input[name=time_limit]').val(
-      UI.parseDuration(problem.settings.limits.TimeLimit),
+    let languages = problem.languages.sort().join();
+    document.getElementsByName('languages')[0].value = languages;
+    problemSettings.languages = languages;
```

Comment:
Aquí podrías hacer todo inline, tipo:
```suggestion
    problemSettings.languages = problem.languages.sort().join();
```
para que ya le pases el dato directo al componente sin tener que manipularlo desde fuera.